setup.cfg

```diff
@@ -0,0 +1,49 @@
+[metadata]
+name = PyJWT
+version = attr: jwt.__version__
+author = Jose Padilla
+author_email = hello@jpadilla.com
+description = JSON Web Token implementation in Python
+license = MIT
+keywords =
+    json
+    jwt
+    security
+    signing
+    token
+    web
+url = https://github.com/jpadilla/pyjwt
+long_description = file: README.rst
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    Natural Language :: English
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Utilities
+
+[options]
+python_requires = >=3.5
+packages = find:
+
+[options.entry_points]
+console_scripts =
+    pyjwt = jwt.__main__:main
+
+[options.extras_require]
+crypto =
+    cryptography >= 1.4
+tests =
+    pytest>=4.0.1,<5.0.0
+    pytest-cov>=2.6.0,<3.0.0
+
```

Comment:
@jdufresne is it possible to do something like this using this setup?

```
EXTRAS_REQUIRE["dev"] = (
    EXTRAS_REQUIRE["tests"]
    + EXTRAS_REQUIRE["cryptography"]
    + EXTRAS_REQUIRE["jwks-client"]
    + ["mypy", "pre-commit"]
)
```

Reply:
As far as I know, values can't be added in `setup.cfg` like in Python code, but we could add a `dev` extras and just copy the entries manually.